tensorflow/c/experimental/filesystem/plugins/cache/cleanup.h

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
We need to ensure that any symbol from this file does not get exported across the interface. Otherwise, we won't be able to separate plugins from core (we won't be able to use a plugin compiled with one version of TF in other versions of TF; we won't be able to use plugins that are compiled with different compiler than TF)